src/ngraph/runtime/dynamic/dynamic_backend.cpp

```diff
@@ -117,130 +117,160 @@ bool runtime::dynamic::DynamicExecutable::call(
     // (1) all shapes;
     // (2) all values of shape-relevant input tensors.
 
-    NGRAPH_CHECK(m_wrapped_function->get_parameters().size() == inputs.size());
+    auto lru = std::make_shared<runtime::LRUCache>(4);
```

Comment:
So the `LRUCache` local object `lru` is created everytime we do `dyn_exec->call`

But we just create the `lru` object to get access to the static (hence global like) objects `m_list` and `m_map`

also the depth of the cache is passed in everytime we create the local `lru` object.
which means this situation is possible:

call(shape=[2]) {
}